.github/workflows/link-check.yaml

```diff
@@ -0,0 +1,49 @@
+name: 'Link Check'
+
+on:
+  workflow_dispatch:
+  push:
+    branches: main
+  pull_request:
+    branches: main
+  schedule:
+    - cron: '34 12 * * *'
+
+jobs:
+  link-check:
+    name: Check GitHub repo links at ${{github.repository}}
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Verify muffet filter wrapper is found
+        run: |
+          WRAPPER_FILENAME="mfw"
+          WRAPPER="./.muffet-filter/$WRAPPER_FILENAME"  
+          if test -f "$WRAPPER"; then
+            echo "using wrapper $WRAPPER"
+          else
+            PROJECT_URL="https://github.com/bhamail/muffet-filter"
+            echo "Missing muffet-filter wrapper file $WRAPPER. Download here: $PROJECT_URL" && exit 1
+          fi
+      - name: Run muffet filter wrapper
+#        run: ./.muffet-filter/mfw --muffet-arg=--verbose --muffet-arg=--accepted-status-codes='200..300,429' --muffet-arg=-f https://github.com/sonatype-nexus-community/contribute.sonatype.com
+#        run: ./.muffet-filter/mfw --muffet-arg=--verbose --muffet-arg=--rate-limit=10 --muffet-arg=--accepted-status-codes='200..300,429' --muffet-arg=-f https://github.com/sonatype-nexus-community/contribute.sonatype.com
+#        TODO Figure out show to remove --one-page-only without swamping the network
```

Comment:
Is all of this needed for the TODO?

Reply:
It's ugly, but sort of my way of keeping track of things to try in order to accomplish the todo